src/main/resources/bootstrap.yaml

```diff
@@ -16,6 +16,8 @@ spring:
         crime-storage-connection-string: storage.crime.connection-string
         reports-email-username: SMTP_USERNAME
         reports-email-password: SMTP_PASSWORD
+        reports-recipients: REPORTS_RECIPIENTS
+
```

Comment:
not sure it is a good idea to have empty line in bootstrap config here :thinking: